src/components/Modal/Modal.tsx

```diff
@@ -46,6 +48,10 @@ export interface ModalProps extends Partial<Props> {
    * http://reactcommunity.org/react-modal/accessibility/#app-element
    */
   appElement?: string | HTMLElement;
+  /**
+   * Additional data that is dispatched with closing hamburger.
+   */
+  trackingClose?: TrackingProps;
```

Comment:
Again, I'd stick to `tracking` here, to keep the naming consistent with other components.